libs/evcs-service/src/test/java/uk/gov/di/ipv/core/library/pact/ContractTest.java

```diff
@@ -0,0 +1,499 @@
+                .headers(
+                        Map.of(
+                                "x-api-key",
+                                EVCS_API_KEY,
+                                "Authorization",
+                                "Bearer " + TEST_EVCS_ACCESS_TOKEN))
+                .willRespondWith()
+                .status(200)
+                .body(getResponseBodyForUserVC())
+                .toPact();
+    }
+
+    private DslPart getResponseBodyForUserVC() {
+        return newJsonBody(
+                        body -> {
+                            body.array(
+                                    "vcs",
+                                    vcs -> {
+                                        vcs.object(
+                                                vc -> {
+                                                    vc.stringType("vc", "vc");
+                                                    vc.stringType("state", "PENDING_RETURN");
+                                                    vc.object(
+                                                            "metadata",
+                                                            metadata -> {
+                                                                metadata.stringType(
+                                                                        "reason", "testing");
+                                                                metadata.numberType(
+                                                                        "timestampMs",
+                                                                        1711721297123L);
+                                                                metadata.stringType(
+                                                                        "txmaEventId",
+                                                                        "1a116fe7-2ff9-4f7c-940d-d55fa7d03d66");
+                                                            });
+                                                });
+                                    });
+                        })
+                .build();
+    }
+
+    @Test
+    @PactTestFor(pactMethod = "validRetrieveVcRequestReturnsUsersVcWith200")
+    void testRetrieveVcRequestReturnsUsersVcWith200(MockServer mockServer) throws Exception {
+        // Under Test
+        EvcsClient evcsClient = new EvcsClient(mockConfigService);
+        var evcsGetUserVCsDto =
+                evcsClient.getUserVcs(TEST_USER_ID, TEST_EVCS_ACCESS_TOKEN, VC_STATES_FOR_QUERY);
+
+        // Assertions
+        assertEquals(1, evcsGetUserVCsDto.vcs().size());
+    }
+
+    @Pact(provider = "EvcsProvider", consumer = "IpvCoreBack")
+    public RequestResponsePact invalidRetrieveVcRequestReturnsWith400(PactDslWithProvider builder) {
+        return builder.given("EVCS client exist")
+                .given("VCS exist")
+                .given("EVC has users VC")
+                .given("test-evcs-api-key is not provided")
```

Comment:
This isn't the right place for this - we are the ones providing the API key, not EVCS.

If you want to omit it, then omit it in the headers below (as you have done). If anything, we should leave in the `test-evcs-api-key is a valid API key`, to prove that the issue is coming from omitting the header.